packages/mariadb-10.11-mroonga/yum/mariadb-10.11-mroonga.spec.in

```diff
@@ -128,6 +128,9 @@ rm -rf $RPM_BUILD_ROOT
 %doc mroonga-doc/*
 
 %changelog
+* Fri Aug 09 2024 Horimoto Yasuhiro <horimoto@clear-code.com> - 14.04-2
+- Rebuild against MariaDB 10.11.9.
```

Comment:
How about commenting that changes have been made to maintain backward compatibility?

Reply:
Umm.
I think that I don't need those information against user of RPM package.
Because we don't fix Mroonga's bug and we don't add a new feature into Mroonga by these modifications.
Mroonga 14.04-2 looks the same as Mroonga 14.04-1 against users.

Reply:
Sure. I agree with you.